[improve][doc] improve pulsar I/O document (#17601)
apache/pulsar: site2/docs/io-quickstart.md

   2020-05-11 20:09:24.533 UTC [1] LOG:  database system is ready to accept connections
   ```

4. Access to PostgreSQL container.

   ```bash
   docker exec -it pulsar-postgres /bin/bash
   ```

> 5. Login PostgreSQL with default username and password:

Reviewer: ```suggestion
5. Log in to PostgreSQL with the default username and password:
```

   ```bash
   psql -U postgres postgres
   ```

> 6. Input following commands to create `pulsar_postgres_jdbc_sink` table:

Reviewer 2: ```suggestion
6. Create a `pulsar_postgres_jdbc_sink` table using the following command:
```

1. Show "action / purpose" first.
2. Below is one command, so remove "s" for "command".

Author: Got it, thanks.
